src/modules/storage/LedgerStorage.ts

```diff
@@ -218,6 +218,29 @@ export class LedgerStorage extends Storages
         return this.query(sql, [height.toString()]);
     }
 
+    /**
+     * Get the highest block height in this Stoa DB
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the block height
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getBlockHeight (): Promise<Height>
+    {
+        return new Promise<Height>((resolve, reject) =>
+        {
+            let sql = `SELECT MAX(height) as height FROM blocks`;
+            this.query(sql, [])
+                .then((row: any[]) =>
+                {
+                    resolve(new Height(BigInt(row[0].height)));
+                })
+                .catch((err) =>
+                {
+                    reject(err);
+                });
+        });
+    }
```

Comment:
If the data does not exist, the value of 'height' will be null.
If the value of 'height' is 'null', an error occurs when 'null' is converted to 'BigInt'.
Then, go to the 'catch' below and call reject(err).
Therefore, if there is no data, 'http status 400' is not delivered and '500' is delivered.
How about the following suggestion?
```suggestion
    public getBlockHeight (): Promise<Height | null>
    {
        return new Promise<Height | null>((resolve, reject) =>
        {
            let sql = `SELECT MAX(height) as height FROM blocks`;
            this.query(sql, [])
                .then((row: any[]) =>
                {
                    if (row[0].height !== null)
                        resolve(new Height(BigInt(row[0].height)));
                    else
                        resolve(null);
                })
                .catch((err) =>
                {
                    reject(err);
                });
        });
    }
```